requirements.txt

```diff
@@ -0,0 +1 @@
+PyYAML == 5.3.1
```

Comment:
I'm surprised that these are exact versions rather than at least some sort of range (>5.3).

Reply:
If for some reason we break with 5.4, we don't want anybody installing the tool in the future to use 5.4. Remember this is a tool that "it should just work", not a library you're incorporating to your project (which you may run tests on, adjust versions, etc.)